tests/package_test.go

```diff
@@ -20,8 +20,8 @@ import (
 func TestPackages(t *testing.T) {
 	// find all packages with *_test.gno files.
 	rootDirs := []string{
-		filepath.Join("..", "examples"),
 		filepath.Join("..", "stdlibs"),
+		// filepath.Join("..", "examples"), // FIXME: consider re-enabling
```

Comment:
If I'm not wrong, this was useless, because we didn't have any `_test.gno` file before this PR.

The issue to keep it, is that `TestPackages` will try to build all the files in the directory, which doesn't allow to use the current `tests/files2/` approach which consists of using `package main`

I let the comment, so we can decide to reenable it later if we decide a new convention to dissociate the `_test.gno` files that should maybe act exactly like `_test.go` files (and follow the package names of the implementation); then, we need how to keep the tests/files2 style files near the implementation (because it's like doc, and valuable for tranpsarency). I don't have a good idea for the convention of such files yet.

Reply:
How about we name file tests as xyz_filetest.go? 
Or, maybe the first line of the file test must start with a special comment like // FILETEST.
I'm looking forward to being able to test example/... modules with regular go xyz_test.go tests (as already done in stdlibs). It makes for better testing, because the filetest way is too focused on output.

Reply:
> How about we name file tests as xyz_filetest.go?
> Or, maybe the first line of the file test must start with a special comment like // FILETEST.

I suggest the first approach or something similar, because filename is enough to know what it is for:
* a human
* an IDE with full disk access
* an extension using `https://gno.land/r/{realm}/` directory listing

---

> I'm looking forward to being able to test example/... modules with regular go xyz_test.go tests (as already done in stdlibs). It makes for better testing, because the filetest way is too focused on output.

Same here, I feel very limited with the current way of testing, but to be honest, I didn't investigated and don't understand how the stdlibs/ is called.

I can investigate later.

Reply:
`gnodev test` will now look for `_filetest.gno` files for current system, and I let the other `testing.T` for another PR.